### ISSUE TITLE
ClearMap Module Not Found

### ISSUE THREAD

**Issue by jzx33** [ClearAnatomics/ClearMap#38]
Running Elastix.py in the ClearMap environment returns a "ModuleNotFoundError: No module named 'ClearMap'" even after following the installation process and activating the environment. 

I am pretty new to using Anaconda, so any help is appreciated.

**Comment by SaibotMagd** [comment]
The path variable isn't correct. Check the "settings.py" file.

**Comment by jzx33** [comment]
Thank you! I checked my "settings.py" file and the path variables, but both the path variables (clearmap_path and elastix_path) led to the cloned ClearMap2 folder(/Programs/ClearMap2/ClearMap and Programs/ClearMap2/ClearMap/External/elastix/build). Is this not the correct path?

**Comment by SaibotMagd** [comment]
Check if environmental path is correct in spyder. "tools->pythonpathmanager"

**Comment by jzx33** [comment]
I checked the environment path of spyder, and it read as /opt/anaconda3/envs/ClearMap/bin/spyder (checked through type -a spyder). I also checked the active environments (conda env list), and ClearMap appeared to be active (/opt/anaconda3/envs/ClearMap had an asterisk next to it). 

I did some further research and saw that those with similar problems had fixed their problem by running the direct path of Spyder, but after activating Spyder from terminal using the full path name, I still got the same error (import ClearMap.IO.IO as io). Others appeared to have had their problem being that Spyder wasn't installed in their environment and another version was running, but given Spyder's existing path, I'm not sure why it still isn't working?

In order to check that this isn't a computer specific problem, I tried to install ClearMap on my other laptop as well, and ran into the same issue, even after taking the extra precautions as mentioned above.

When I tried to run ( from ClearMap.Environment import *) in the Python Console, I got this error:

clang: error: unsupported option '-fopenmp'
Traceback (most recent call last):
  File "/opt/anaconda3/envs/ClearMap/lib/python3.7/distutils/unixccompiler.py", line 118, in _compile
    extra_postargs)
  File "/opt/anaconda3/envs/ClearMap/lib/python3.7/distutils/ccompiler.py", line 910, in spawn
    spawn(cmd, dry_run=self.dry_run)
  File "/opt/anaconda3/envs/ClearMap/lib/python3.7/distutils/spawn.py", line 36, in spawn
    _spawn_posix(cmd, search_path, dry_run=dry_run)
  File "/opt/anaconda3/envs/ClearMap/lib/python3.7/distutils/spawn.py", line 159, in _spawn_posix
    % (cmd, exit_status))
distutils.errors.DistutilsExecError: command 'gcc' failed with exit status 1


I saw that this error occurred before, and I'll try to follow the instructions listed to fix it, and will update my results. Thank you for your help!
